### DISCUSSION TITLE
Subscribe Between Threads

### DISCUSSION THREAD

**Discussion by grijjly** [uNetworking/uWebSockets.js#947] [Q&A]
when i use multithread example, when ws subscribe to channel, other threads connections with same subscribtion get the data ?

**Comment by uNetworkingAB** [comment]
No that's not how it works. You can read the user manual about this

**Reply by SupertigerDev** [reply]
@uNetworkingAB could you link me to the manual at that section.

In socket.io, i would need to use a redis adapter to make sure all of my clients, connected to any worker instance of the website server will get the date from another instance.

How is this done? Is it done automatically?
